docs/tutorials/embedding/OutOfSampleEmbed.ipynb

```diff
@@ -274,8 +326,41 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot_oos(X_hat, w[0], labels, oos_labels=oos_labels, title=\"Out Latent Predictions\")\n",
-    "plot_oos(Y_hat, w[1], labels, oos_labels=oos_labels, title=\"In Latent Predictions\")"
+    "plot_oos(X_hat_ase, w_ase[0], labels, oos_labels=oos_labels, title=\"ASE Out Latent Predictions\")\n",
+    "plot_oos(Y_hat_ase, w_ase[1], labels, oos_labels=oos_labels, title=\"ASE In Latent Predictions\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Fit our directed graph\n",
+    "X_hat_lse, Y_hat_lse = lse.fit_transform(A)\n",
+    "\n",
+    "# predicted latent positions\n",
+    "w_lse = lse.transform(a)\n",
+    "print(f\"output of `ase.transform(a)` is {type(w_lse)}\", \"\\n\")\n",
+    "print(f\"out latent positions: \\n{w_lse[0]}\\n\")\n",
+    "print(f\"in latent positions: \\n{w_lse[1]}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plotting directed LSE latent predictions"
```

Comment:
Weird question, but can you use a different random seed? The `LSE Out Latent Predictions` on my tutorial run has an out-of-sample latent position which is weirdly far away from the mean, so not very good as an explanatory image